web/views/api/index_view.ex

```diff
@@ -0,0 +1,8 @@
+defmodule HexWeb.API.IndexView do
+  use HexWeb.Web, :view
+
+  def render("index." <> _format, _assigns) do
+    %{packages_url: api_package_url(Endpoint, :index),
+      documentation_url: "http://docs.hexpm.apiary.io"}
+  end
```

Comment:
@ericmj we could also add "GitHub API-style" placeholder urls (api.github.com) so this whole response would look like this:

```
{
  "packages_url":        "http://localhost:4000/api/packages",
  "package_url":         "http://localhost:4000/api/packages/{name}",
  "package_release_url": "http://localhost:4000/api/packages/{name}/releases/{version}",
  "package_owners_url":  "http://localhost:4000/api/packages/{name}/owners",
  "user_url":            "http://localhost:4000/api/users/{name}",
  "documentation_url":   "http://docs.hexpm.apiary.io"
}
```